src/styles/main.css

```
@@ -1,10 +1,22 @@
 html body {
```

you probably meant for this to be
```suggestion
html,
body {
```